homeassistant/components/shelly/binary_sensor.py

```diff
@@ -64,7 +64,10 @@
         icon="mdi:update",
         default_enabled=False,
         path="update/has_update",
-        attributes={"description": "available version:", "path": "update/new_version"},
+        attributes=[
+            {"description": "released_version", "path": "update/new_version"},
```

Comment:
Should the description be latest_stable?

Reply:
`latest_stable_version` would work for everybody ?